README.md

```diff
@@ -13,3 +13,7 @@ Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
 The page will reload when you make changes.\
 You may also see any lint errors in the console.
+
+This project using:
+react-icons
+react-bootstrap
```

Comment:
Since this is listed in the package.json, it is a bit redundant to add it into the readme. Unless you particularly want to highlight it again!